app/component/ItineraryCircleLine.js

```diff
@@ -15,6 +16,18 @@ class ItineraryCircleLine extends React.Component {
     color: PropTypes.string,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      isMounted: false,
```

Comment:
No need to have isMounted. Instead, you should do an import('...png').then(imageUrl => this.setState({imageUrl})). Also state can be initialised directly using a class property initializer.